ui/app/adapters/pki/issuer.js

```diff
@@ -13,15 +13,21 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
   }
 
   urlForQuery(backend, id) {
-    let url = `${this.buildURL()}/${encodePath(backend)}/issuers`;
+    const baseUrl = `${this.buildURL()}/${encodePath(backend)}`;
     if (id) {
-      url = url + '/' + encodePath(id);
+      return `${baseUrl}/issuer/${encodePath(id)}`;
+    } else {
+      return `${baseUrl}/issuers`;
     }
-    return url;
   }
 
   query(store, type, query) {
     const { backend, id } = query;
```

Comment:
do we need `id` and `this.optionsForQuery` here since below is the method used for querying a single record?